cmd/m3/setup.go

```diff
@@ -32,6 +34,24 @@ var setupCmd = cli.Command{
 }
 
 func setupDefCmd(ctx *cli.Context) error {
-	cluster.SetupM3()
-	return nil
+	name := ctx.String("name")
+	email := ctx.String("email")
+	if name == "" && ctx.Args().Get(0) != "" {
+		name = ctx.Args().Get(0)
+	}
+	if email == "" && ctx.Args().Get(1) != "" {
+		email = ctx.Args().Get(1)
+	}
+
+	if name == "" {
+		fmt.Println("An admin name is needed")
+		return errMissingArguments
+	}
+
+	if email == "" {
+		fmt.Println("An admin email is needed")
+		return errMissingArguments
+	}
+	cluster.SetupM3(name, email)
```

Comment:
oh lol it's being declared twice

Reply:
lol, I saw your suggestion and I only thought "yeah retuning the error makes sense" hahaha